source/ingress.go

```diff
@@ -167,12 +175,17 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 			continue
 		}
 
+		ttl, err := getTTLFromAnnotations(ing.Annotations)
+		if err != nil {
+			log.Warn(err)
+		}
```

Comment:
Since you continue the loop here we should set `ttl` to something sane, e.g. `300`.

Reply:
i think it is okay to leave it as it is here, as this should be handled in providers (if they care about TTL field in endpoint)